components/brave_rewards/resources/extension/brave_rewards/utils.ts

```diff
@@ -58,3 +58,37 @@ export const getGrant = (grant?: RewardsExtension.GrantInfo) => {
 
   return grant
 }
+
+export const setBadgeText = (state: RewardsExtension.State, verified: boolean = false, tabId: number = -1) => {
```

Comment:
nit: this is not a generic 'util', plus it's best to keep things that are unrelated in separate files, in case we do chunking or asynchronous dependency imports (`import()`). Plus, it uses app state as a param. I suggest this is just put in its own module `setBadgeText` or, if you want to be more generic then, `browserAction.ts`.

Reply:
done